examples/socket.io-chat-app/index.js

```diff
@@ -0,0 +1,20 @@
+const micro = require('micro'),
+      fs = require('fs');
+
+const server = micro(async (req, res) => {
+  const html = fs.readFileSync(__dirname + '/index.html');
```

Comment:
You don't want to read the index contents every time the route is hit, specially not *sync*, this will block other incoming connections